File: JH/22.01.20/10992.py
N = int(input())
for i in range(1, N+1):
  if i == 1:
    print(' '*(N-i)+'*'*i)
  elif i == N:
    print('*'*(i*2-1))
  else:
    print(' '*(N-i)+'*'+ ' '*(i*2-3)+'*')

File: Lami/02.05/10992.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

function solution() {
  const line = Number(input[0]);
  let cnt = 1;
  for (let i = 1; i < line; i++) {
    str = '';
    if (i === 1) {
      console.log(' '.repeat(line - i) + '*');
    }
    if (i >= 2) {
      str = '*' + ' '.repeat(cnt) + '*';
      console.log(' '.repeat(line - i) + str);
      cnt += 2;
    }
  }
  console.log('*'.repeat(2 * line - 1));
}

solution();
